lib/internal/url.js

```diff
@@ -1392,11 +1393,11 @@ function getPathFromURLPosix(url) {
   return decodeURIComponent(pathname);
 }
 
-function getPathFromURL(path) {
-  if (path == null || !path[searchParams] ||
-      !path[searchParams][searchParams]) {
-    return path;
-  }
+function fileURLToPath(path) {
+  if (typeof path === 'string')
+    path = new URL(path);
+  else if (!(path instanceof URL))
+    throw new ERR_INVALID_ARG_TYPE('path', 'string or URL', path);
```

Comment:
Change `'string or URL'` to `['string', 'URL']`.

Reply:
Ah, nice, thanks